src/ui/Base/QueryBuilder.ts

```diff
@@ -320,6 +320,10 @@ export class QueryBuilder {
    * This parameter is normally controlled by {@link SearchInterface.options.allowEmptyQuery} option.
    */
   public allowQueriesWithoutKeywords: boolean;
+  /**
+   * Specify the information about the user who's actions shall be checked in the query.
+   */
+  public userActions: IUserActions = {};
```

Comment:
I would adjust the property name to `userActionsRequest` here as well

Reply:
If you check the build method, apart from the expressions (that are single or two letters when sent), we are using exactly the same prop name as the field we send to the searchApi, so I'm not sure we should change it,

Reply:
Ah, in that case the interface should be good enough. Thanks Louis,